### DISCUSSION TITLE
I cant get the plane in my MSFS hangar?

### DISCUSSION THREAD

**Discussion by grudziadz-gkm** [flybywiresim/aircraft#9449] [Help]
### Aircraft Version

Stable

### Build info

```json
ssaaaaaaaaaaaaaaaaaaaaaa
```

### Describe the bug

Doesnt show the plane in my hangar

### Expected behavior

the plane to be in my MSFS hangar

### Steps to reproduce

downloaded the plane but didint show in my hangar but in FBW installer (Fly by wire Installer) it shows it has been downloaded! Im confused? whats the issue ( i have the same problem with my a320 FBW)

### References (optional)

w

### Additional info (optional)

w

### Discord Username (optional)

captainaryan_

**Comment by alepouna** [comment]
Hello, you probably have installed the aircraft in the wrong community folder. 

We recommend you reach out via Discord for support first as this is a non-aircraft issue. I have converted this issue to a discussion in case you can't use Discord.
